### DISCUSSION TITLE
在vite中如何动态切换主题

### DISCUSSION THREAD

**Discussion by pincman** [rsuite/rsuite#1610] [Q&A]
看到官方文档有一个动态切换主题的插件，想问一下在vite中如何使用这个功能

**Comment by SevenOutman** [comment]
RSuite v4 并不支持动态切换主题，文档中提到的插件是“编译多份主题样式”，并非动态切换主题。若要动态切换主题，可编译多份主题样式文件，通过修改 `<link>` 样式表元素的地址来实现。
